doc/contributing/submitting-changes.chapter.md

```diff
@@ -83,6 +83,62 @@ If a security fix applies to both master and a stable release then, similar to r
 
 Critical security fixes may by-pass the staging branches and be delivered directly to release branches such as `master` and `release-*`.
 
+## Writing package tests {#submitting-changes-package-tests}
+
+Tests are important to improve quality and make reviews and automatic updates easy.
+
+Nix package tests are a lightweight alternative to NixOS module tests. They can be used to create simple integration tests for packages while the module tests are used to test services or programs with a graphical user interface on a NixOS VM.
+
+This is an example using the `phoronix-test-suite` package with the current best practice.
+
+Add this to the package definition:
+
+```
```

Comment:
Please include the syntax language

```suggestion
```nix
```

Reply:
is that actually used in the end product?

Reply:
Yes, pandoc preserves the class as `language` attribute of code listing in Docbook and `docbook-xsl` then passes that as class on the `pre` elemenr, which highlight.js syntax highlighter uses to choose syntax definition.